native/src/main/java/io/ballerina/stdlib/grpc/Message.java

```diff
@@ -153,6 +164,17 @@ public Message(
             bMessage = bArray;
         } else if (type.getTag() == TypeTags.DECIMAL_TAG) {
             bMessage = ValueCreator.createDecimalValue(new BigDecimal(0));
+        } else if (type.getTag() == TypeTags.MAP_TAG) {
```

Comment:
Shall we add a comment for each case here with the mapping? For example `// TypeTags.INTERSECTION_TAG for Timestamp` likewise. It will improve the readability.

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/368/commits/088146bd664cf598947fa2dae8cf6be93ac5ca50